### ISSUE TITLE
"Date of surgery" should allow multiple values

### ISSUE THREAD

**Issue by apdavison** [INCF/neuroshapes#138]
In core/subject/v0.1.0.json, `nsg:dateOfSurgery` has `"maxCount": 1`

For subjects with chronically implanted electrodes, or multiple electrode arrays, it is possible to have more than one surgery, on different days.

**Comment by MFSY** [comment]
Hi,
This an interesting point. If many dates of surgery are possible for a given subject than a  nsg:dateOfSurgery value should belong to the experiment activity which performs the "surgery" I think. Multiple dates of surgery within an array may lead to confusion .

Okay for nsg:disease and nsg:treatment.

What do you @annakristinkaufmann  ?


on   the nsg:dateOfSurgery case,  case

**Comment by apdavison** [comment]
ok, I agree. That suggests removing `nsg:dateOfSurgery` altogether, then?

**Comment by annakristinkaufmann** [comment]
I agree with you, @MFSY ! The reason we'd included it in the first place was that the nsg:dateOfSacrifice property couldn't be used for the date of human tissue extraction, so we've included nsg:dateOfSurgery as an additional property shape (since we did not want to model the surgery activity per se). We could also make this activity explicit and include it in there.
